docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,283 @@
+         mkdir -p $CONDA_PREFIX/etc/conda/activate.d \
+         && mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d \
+         && echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/'>\
+         $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh && \
+         echo 'unset LD_LIBRARY_PATH'>\
+         $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+
+         # Reinitialize environment
+         conda activate base && conda activate pkd-training
+
+   .. tab:: Mac 
+    
+      Platform:
+         - MacOS Intel Chip or MacOS M1/M2 chip
+      Prerequisite:
+         - `Git <https://github.com/git-guides/install-git>`_
+         - `Miniforge <https://github.com/conda-forge/miniforge#homebrew>`_
+
+         On macOS, you can install miniforge with Homebrew by running
+
+         .. code-block:: bash
+            :linenos:
+
+            brew install miniforge
+
+      Install requirements
+
+         .. code-block:: bash
+            :linenos:
+
+            # Clone PeekingDuck repository
+            git clone <TODO: PeekdingDuck repository>
+            cd PeekdingDuck
+
+            # Create conda environment & activate
+            conda env create -f peekingduck/training/requirements_mac.yaml
+            conda activate pkd-training
+
+      Install TensorFlow
+
+         .. code-block:: bash
+            :linenos:
+            
+            bash .peekingduck/training/scripts/install_tensorflow_macos.sh
+
+.. _setting-up-weights-and-biases:
+
+Setting up Weights & Biases
+---------------------------
+
+Follow the steps below to get configure weights & biases:
+
+1. `Sign up <https://wandb.ai/site>`_ for a free account and then login to your wandb account.
+2. (If not yet installed via requirements.txt) Pip install the wandb library on your machine in a Python 3 environment.
+3. Login to the wandb library on your machine. You will find your API key `here <https://wandb.ai/authorize>`_
+
+.. code-block:: bash
+   :linenos:
+   
+   pip install wandb
+
+   # Login to weights & biases
+   wandb login
+
+Weights & Biases comes out of the box with the training pipeline but in the case where you do not want to use weights & biases you can disable it as such.
+
+.. code-block:: bash
+   :linenos:
+   
+   # Disable weights & biases
+   wandb disable
+
+You may refer to the `official guide <https://docs.wandb.ai/quickstart>`_ for setting up an account.
+
+
+========
+Test Run
+========
+
+You can test the training pipeline with the default :mod:`cifar10` dataset using the following commands in terminal:
+
+.. code-block:: bash
+   :linenos:
+   
+   # Use the default configurations to test
+
+   cd PeekingDuck
+
+   # Tensorflow
+   python ./peekingduck/training/main.py debug=True framework=tensorflow
+
+   # Pytorch
+   python ./peekingduck/training/main.py debug=True framework=pytorch
+
+View the results of each run at the specified output folder directory: :mod:`\./Peekingduck/outputs/\<PROJECT_NAME\>/\<DATE_TIME\>`
```

Comment:
The `<PROJECT_NAME>` is used here but not referenced / introduced anywhere up above.
This creates confusion as the reader will wonder what is this `<PROJECT_NAME>`.
Need to rewrite above to make sure the flow is coherent and the information is complete. If need to use project name, then specify and set it before arriving at this line.